docs/kit/operation-view/installation_guide.md

```diff
@@ -0,0 +1,116 @@
+Installation Guide for Tractus-X EDC
+==================
+This guide describes how to install the [Tractus-X EDC](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/charts/tractusx-connector) via helm chart with Hashicorp Vault + PostgreSQL.
+
+Prerequisite
+------------
+If you do not have preexisting installations of PostgreSQL or HashiCorp Vault, you will need to configure them first.
```

Comment:
A more detailed list of prerequsites like in this [sample](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/docs/samples/example-dataspace) would be great.
You should be able to complete the installation if you have everything listed in the prerequisite, including all used tools and requirements.